### DISCUSSION TITLE
How to assign anonymous commit author to profil github category ?

### DISCUSSION THREAD

**Discussion by MatthDfx** [community/community#43477] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,
I had an email address problem between my github and my local git and all my commits on common projects are signed by my first and last name but unrelated to my github profile. How to link the two? Thanks

**Comment by genidma** [comment]
1. How did you authenticate in the first place? You may have to re-authenticate? I am thinking.
2. Also, once re authenticated, would it be advisable to delete the existing repository, create a new repository and push your code to the new repository (with the proper authentication)?

Thx
